lib/matplotlib/axes/_base.py

```diff
@@ -1316,7 +1324,29 @@ def get_adjustable(self):
 
     def set_adjustable(self, adjustable):
         """
-        ACCEPTS: [ 'box' | 'datalim' | 'box-forced']
+        Define which parameter the axes will change to achieve a given aspect.
+
+        Possible values are:
+
+        ============   =====================================
+        value          description
+        ============   =====================================
+        'box'          change physical size of axes
+        'datalim'      change xlim or ylim
+        'box-forced'   same as 'box', but axes can be shared
+        ============   =====================================
+
+        'box' does not allow axes sharing, as this can cause
+        unintended side effect. For cases when sharing axes is
+        fine, use 'box-forced'.
+
+        ..
+            ACCEPTS: [ 'box' | 'datalim' | 'box-forced']
+
+        See Also
+        --------
+        matplotlib.axes.Axes.set_aspect
```

Comment:
link? (I didn't check the rendered version, maybe the links come through in See Also?